src/igvfd/schemas/changelogs/in_vitro_system.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`in_vitro_system.json`*
 
+### Schema version 9
+
+* Remove `taxa` from `required`.
```

Comment:
I think it woudl be nice to indicate the property is not only not required but also calculated